web/src/main/webapp/resources/js/atsy-new-application-popup.js

```diff
@@ -0,0 +1,46 @@
+var positionDefaultValueInEng = 'Please choose a position';
```

Comment:
Wrap your whole code into an IFFE (Immediately Invoked Function Expression) to prevent pollution of global scope.

`(function(){ /* code */ }());`